src/domotica/domotica_fastclock.c

```diff
@@ -0,0 +1,114 @@
+/*
+ * @file domotica_fastclock.c
+ * @brief Loconet Domotica Module - listen to the fast clock
+ *
+ * \copyright Copyright 2017 /Dev. All rights reserved.
+ * \license This project is released under MIT license.
+ *
+ * @author Jan Martijn van der Werf
+ */
+
+#include "domotica_fastclock.h"
+
+// ----------------------------------------------------------------------------
+// Array of timestamps to react on. If the LNCV number in the array equals 0,
+// it is an empty slot.
+typedef struct {
+  uint16_t timestamp;
```

Comment:
Commentaar toegevoegd. Ook in domotica.h. De variabele representeert echt een timestamp (alleen niet tot op de seconde nauwkeurig), en niet een pattern.